extensions/exposition/source/HTTP/headers.ts

```diff
@@ -0,0 +1,53 @@
+import type * as http from 'node:http'
+
+export function logHeaders (headers: http.IncomingMessage['headers']): void {
+  if (headers instanceof Headers)
+    for (const header of headers.entries())
+      console.debug(`${header[0]}: ${header[0] in ALLOWED_HEADERS ? header[1] : '...'}`)
+}
+
+enum ALLOWED_HEADERS {
```

Comment:
This is really incredible.

1. Why do we interested in these specific headers?
2. Why is around half of the list composed of response headers that can't be part of the request?